cle/backends/relocations/peamd64.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.peamd64')
+
+arch = 'peAMD64'
```

Comment:
Then I guess get_relocation would be moved from the base into the specific module so we wouldn't have to namespace the arches like that. The "scraping" code could stay in the base, though.